presto-spi/src/main/java/com/facebook/presto/spi/ConnectorPageSink.java

```diff
@@ -16,8 +16,18 @@
 import com.facebook.presto.spi.block.Block;
 
 public interface ConnectorPageSink
+        extends AutoCloseable
```

Comment:
The other classes like `ConnectorSplitSource` extend `Closeable` and do not throw checked exceptions. I think we should keep the same constraints here.